<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCourses</class>
 <widget class="QDialog" name="addCourses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>193</width>
     <height>204</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="basicInfo">
    <item row="5" column="1">
     <widget class="QLineEdit" name="nowSeleted"/>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="ifMust">
      <property name="text">
       <string>必修</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>授课教师</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="limit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>是否必修</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>课程名称</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学分</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>限选</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="teacherName"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="courseName"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>已选</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="coursePoint"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>意愿值</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="intension"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>添加课程</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="courseSchedule">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>791</width>
     <height>391</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>12</number>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <row>
    <property name="text">
     <string>8:00-8:50</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9:00-9:50</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10:10-11:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11:10-12:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13:00-13:50</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14:00-14:50</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15:10-16:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16:10-17:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17:10-18:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18:40-19:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>19:40-20:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>20:40-21:30</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>星期一</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期二</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期三</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期四</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期五</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期六</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期日</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
